extensions/wikia/ImageServing/drivers/ImageServingDriverMainNS.class.php

```diff
@@ -189,19 +189,19 @@ protected function getImagesPopularity( Array $imageNames, $limit ) {
 
 		$imageNames = array_values( $imageNames );
 
-		// MostimagesInContentPage generates daily reports with images
+		// MostLinkedFilesInContentPage generates daily reports with images
 		// that are included at least twice in article in content namespaces
 		$ret = $this->db->select(
 			'querycache',
 			[ 'qc_title as image', 'qc_value as popularity' ],
 			[
-				'qc_type' => 'MostimagesInContent',
+				'qc_type' => 'MostLinkedFilesInContent',
```

Comment:
After the release there will be no data with `'qc_type' = 'MostLinkedFilesInContent'`. Cron job regenerates this data via `rebuildSpecialPages` script every day.